e2e/workloads/deploy.go

```diff
@@ -57,6 +57,14 @@ func (w Deployment) GetBranch() string {
 	return w.Branch
 }
 
+func (w Deployment) GetSelectResource() string {
+	return "deployment"
```

Comment:
Should we use "apps/v1/deployments" like kubevirt or recipe expects this specific value?   If this is a specific value we need to document it here, and later add a constant in the recipe package to make this type safe.